Modules/cjkcodecs/multibytecodec.h

```diff
@@ -94,6 +94,12 @@ typedef struct {
     _MultibyteStatefulEncoder_HEAD
 } MultibyteIncrementalEncoderObject;
 
+typedef struct {
+    unsigned char pending[MAXENCPENDING];
+    Py_ssize_t pendingsize;
+    MultibyteCodec_State codecstate;
+} MultibyteIncrementalEncoderState;
```

Comment:
This struct is used from only two place in single source file.
Move this to the file.